classes/ActionScheduler_ListTable.php

```diff
@@ -0,0 +1,354 @@
+<?php
+
+/**
+ * Implements the admin view of the actions.
+ */
+class ActionScheduler_ListTable extends PP_List_Table {
+	/**
+	 * The package name. It is used also as the domain for the translations
+	 */
+	protected $package = 'action-scheduler';
+
+	/**
+	 * Columns to show (name => label). The label is automatically
+	 * translated before rendering
+	 */
+	protected $columns = array(
+		'hook'   => 'Hook',
+		'status' => 'Status',
+		'args'   => 'Arguments',
+		'group'  => 'Group',
+		'recurrence' => 'Recurrence',
+		'scheduled'  => 'Scheduled Date',
+		'claim_id'   => 'Claim ID',
+		'comments'   => 'Log',
```

Comment:
These strings should all be made available for translation and passed through i18n functions.

Reply:
The library (PP_Table_List) would translate any string automatically.

Reply:
https://github.com/Prospress/action-scheduler/blob/issue_85/vendor/pp-list-table/src/class-pp-list-table.php#L97-L99

Should I make the translation implicit in the code?